package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.4.0",
     "@testing-library/user-event": "^7.2.1",
+    "big.js": "^5.2.2",
```

Comment:
This is not the correct dependency, the dependency listed in the [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896094/) is [This one](https://www.npmjs.com/package/big-js); that is probably why you are getting that stickler error since you are loading a different dependency from the one indicated.